lib/exercism/node.rb

```diff
@@ -0,0 +1,55 @@
+ # {:path=>"README.txt",
+ #    :mode=>"100644",
+ #    :type=>"blob",
+ #    :sha=>"e69de29bb2d1d6434b8b29ae775ad8c2e48c5391",
+ #    :size=>0,
+ #    :url=>""}
+
+ # CONVERTED TO
+
+ # "[{\"id\":\"README.txt\",\"parent\":\"#\",\"icon\":\"file\",
+ #    \"data\":{\"sha\":\"e69de29bb2d1d6434b8b29ae775ad8c2e48c5391\",\"type\":\"file\"}}]"
```

Comment:
Line is too long. [90/80]<br>Incorrect indentation detected (column 1 instead of 0).